integration/common.go

```diff
@@ -217,18 +217,18 @@ func CreateCredentialApplicationJWT(credApplication credApplicationParams, crede
 	return signed.String(), nil
 }
 
-type definitionParams struct {
+type presentationRequestParams struct {
 	Author    string
 	AuthorKID string
 }
 
-func CreatePresentationDefinition(params definitionParams) (string, error) {
-	logrus.Println("\n\nCreate our Presentation Definition:")
-	definitionJSON, err := resolveTemplate(params, "presentation-definition-input.json")
+func CreatePresentationRequest(params presentationRequestParams) (string, error) {
```

Comment:
I'm not sure about this. I see a value in storing a presentation definition, and then generating a presentation request separately, namely because they might have different wrapping containers.

I would be open to changing the endpoints to `CreatePresentationDefintion` and `GetPresentationRequest` but I'd really rather see this PR revert the Presentation Definition changes and add a new `GetPresentationRequest` endpoint which returns a signed Presentation Definition.

Reply:
> What I'm doing here is just naming things correctly, which I believe is important to do. Adding a new endpoint wouldn't fix the incorrect names.

Creating a Presentation Definition doesn't take in a signed request. It should store the Presentation Definition object.

> Can you elaborate the value of storing a presentation definition separate from the request in which it's going to be transported in?

Yes. Presentations are often created for specific parties. Meaning they have an audience (`aud` in jwt land), and expiration (`exp`) to denote how long they're valid for. Sometimes we can create longer-lived presentations, and presentations that don't have a specific audience. This design boxes us into that last case.